File: blacklists.js
const fs = require("fs");
const prompt = require("prompt-sync")();
var path = prompt("please input your file name: ");
if (path == "blocked.txt") {
    let name = prompt("please enter the name: ");
    let phoneNumber = prompt("please enter the phone number: ");
    checkBlacklist(name, phoneNumber);
} else {
    console.log("sorry couldn't find your file name, check again.");
}

function initialize(path) {
    let blacklist = fs.readFileSync(path).toString();
    return blacklist;
}

function checkBlacklist(name, phoneNumber) {
    let blacklist = initialize(path);
    let args = name.toLowerCase() + " " + phoneNumber.toLowerCase();
    if (blacklist.includes(args)) {
        console.log("It is on Blacklist");
    } else {
        console.log("Not in Blacklist");
    }
}

File: blacklists.py
def initialize(blacklist):
    if blacklist == "blocked.txt":
        with open("blocked.txt", "r") as file:
            blist = file.read()
            return blist

def check_blacklist(name, phone_number):
    blist = initialize(blacklist)
    prompt = name.lower() + " " + str(phone_number)
    if prompt in blist.lower():
        blacklisted = True
        print("It is on blacklist")
    else:
        blacklisted = False
        print("not in blacklist")
    return blacklisted

while True:
    blacklist = input("please input your file name: ")
    if blacklist == "blocked.txt":
        name = input("Please input the name: ")
        phone_number = int(input("please input the phone number: "))
        check_blacklist(name, phone_number)
        break
    print("sorry couldn't find your file name, check again.")